org.doom2d.forever.yml

```diff
@@ -0,0 +1,96 @@
+app-id: org.doom2d.forever
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+build-options:
+  append-path: /usr/lib/sdk/freepascal/bin
+  env:
+    PPC_CONFIG_PATH: /usr/lib/sdk/freepascal/etc
+    FPCDIR: /usr/lib/sdk/freepascal/fpcsrc
+    FPC_DIR: /usr/lib/sdk/freepascal/fpcsrc
+default-branch: stable
+command: Doom2DF
+cleanup:
+  - ['*']
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+modules:
+  - name: gme
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.gz
+        sha256: 626e8a104e0dadd10ef6519a67aca880c7b40f81471659f1935b61754e12fc7b
+        x-checker-data:
+          type: anitya
+          project-id: 866
+          url-template: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-$version.tar.gz
+  - name: libmodplug
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/modplug-xmms/libmodplug-0.8.9.0.tar.gz
+        sha256: 457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de
+        x-checker-data:
+            type: anitya
+            project-id: 5669
+            url-template: https://downloads.sourceforge.net/modplug-xmms/libmodplug-$version.tar.gz
+  - name: libxmp
+    sources:
+      - type: archive
+        url: https://github.com/libxmp/libxmp/releases/download/libxmp-4.5.0/libxmp-4.5.0.tar.gz
+        sha256: 7847d262112d14e8442f44e5ac6ed9ddbca54c251284720b563c852b31f26e75
+  - name: enet
+    sources:
+    - type: git
+      url: https://github.com/lsalzman/enet.git
+      branch: master
+    - type: script
+      dest-filename: autogen.sh
+      commands:
+        - autoreconf -ifv
+  - name: d2df-sdl
+    sources:
+    - type: git
+      url: https://github.com/Challenge9/doom2d-forever.git
+      branch: master
```

Comment:
This should be a commit or a tag. NOT a branch, ever.

Reply:
Makes sense. I'll use a tag.